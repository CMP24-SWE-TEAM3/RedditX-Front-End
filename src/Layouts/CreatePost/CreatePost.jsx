// imports
import React from 'react';
import {Container} from "./CreatePost.styled";
import Post  from "Components/Post/Post";
import ImageButton   from "Components/ImageButton/ImageButton";
import LinkButton from "Components/LinkButton/LinkButton";
import ProfilePostButton from "Components/ProfilePostButton/ProfilePostButton";
/**
 * Component that shows input field to entry text , Images  and link  etc....
 *
 * @returns {React.Component}
 */
const CreatePost = () => {
    return (
        <Container>
            <ProfilePostButton/>
            <Post/>
            <ImageButton/>
            <LinkButton/>
        </Container>
    );
};

export default CreatePost;
