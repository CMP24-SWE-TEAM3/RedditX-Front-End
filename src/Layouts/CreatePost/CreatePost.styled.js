import styled from "styled-components";

export const Container = styled.div`
  display: flex;
  margin-bottom: 16px;
  padding: 8px;
  border-radius: 4px;
  border: 1px solid ${({theme}) => theme.background.primary};
  //vertical-align: baseline;
  background-color: ${({theme})=>theme.background.primary};
  @media only screen and (max-width: 961px) {
    width: 100%;
  }
`