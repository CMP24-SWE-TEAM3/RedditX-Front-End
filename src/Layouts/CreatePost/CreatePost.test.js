import {shallow} from 'enzyme';

// import Components
import CreatePost from "./CreatePost";

describe("Create Post Component", () => {
    it("this is a test for Create Post Component", () => {
        expect(shallow(<CreatePost/>)).toMatchSnapshot();

    });
});