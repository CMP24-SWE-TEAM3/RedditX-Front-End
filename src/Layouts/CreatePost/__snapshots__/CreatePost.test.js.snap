// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Post Component this is a test for Create Post Component 1`] = `ShallowWrapper {}`;
